api/envoy/config/filter/network/dubbo_proxy/v2alpha1/dubbo_proxy.proto

```diff
@@ -1,28 +1,55 @@
 syntax = "proto3";
 
-package envoy.extensions.filters.network.dubbo_proxy.v2alpha1;
-option java_package = "io.envoyproxy.envoy.extensions.filters.network.dubbo_proxy.v2alpha1";
+package envoy.config.filter.network.dubbo_proxy.v2alpha1;
+option java_package = "io.envoyproxy.envoy.config.filter.network.dubbo_proxy.v2alpha1";
 option java_multiple_files = true;
 option go_package = "v2";
 
+import "envoy/config/filter/network/dubbo_proxy/v2alpha1/route.proto";
+
+import "google/protobuf/struct.proto";
+
 import "validate/validate.proto";
+import "gogoproto/gogo.proto";
 
 // [#protodoc-title: Dubbo Proxy]
 // Dubbo Proxy filter configuration.
-
 message DubboProxy {
   // The human readable prefix to use when emitting statistics.
   string stat_prefix = 1 [(validate.rules).string.min_bytes = 1];
 
   // Configure the protocol used.
-  enum ProtocolType {
-    Dubbo = 0; // the default protocol.
-  }
   ProtocolType protocol_type = 2 [(validate.rules).enum.defined_only = true];
 
   // Configure the serialization protocol used.
-  enum SerializationType {
-    Hessian2 = 0; // the default serialization protocol.
-  }
   SerializationType serialization_type = 3 [(validate.rules).enum.defined_only = true];
+
+  // The route table for the connection manager is static and is specified in this property.
+  repeated RouteConfiguration route_config = 4;
+
+  // A list of individual Dubbo filters that make up the filter chain for requests made to the
+  // Dubbo proxy. Order matters as the filters are processed sequentially. For backwards
+  // compatibility, if no dubbo_filters are specified, a default Dubbo router filter
+  // (`envoy.filters.dubbo.router`) is used.
+  repeated DubboFilter dubbo_filters = 5;
 }
+
+enum ProtocolType {
+  Dubbo = 0; // the default protocol.
+}
+
+enum SerializationType {
+  Hessian2 = 0; // the default serialization protocol.
+}
+
+// DubboFilter configures a Dubbo filter.
+// [#comment:next free field: 3]
+message DubboFilter {
+  // The name of the filter to instantiate. The name must match a supported
+  // filter.
+  string name = 1 [(validate.rules).string.min_bytes = 1];
+
+  // Filter specific configuration which depends on the filter being
+  // instantiated. See the supported filters for further documentation.
+  google.protobuf.Struct config = 2;
```

Comment:
I think we can do google.protobuf.Any directly, see #4475 for details.

Reply:
Yes, very good, it has been modified.